apps/examples/testcase/le_tc/filesystem/tc_fs_mops.c

```diff
@@ -160,11 +147,15 @@ static void tc_fs_mops_mount(const char *filesystemtype)
 	if (strcmp(filesystemtype, "tmpfs") == 0) {
 		ret = umount(TMPFS_TEST_MOUNTPOINT);
 		ret = mount(NULL, TMPFS_TEST_MOUNTPOINT, "tmpfs", 0, NULL);
-	} else if (strcmp(filesystemtype, "romfs") == 0) {
+	}
+#ifndef CONFIG_BUILD_PROTECTED
+	else if (strcmp(filesystemtype, "romfs") == 0) {
```

Comment:
This violates the Tizen coding rule M07.

You need to modify it like this:

#ifndef CONFIG_BUILD_PROTECTED
    } else if (strcmp(filesystemtype, "romfs") == 0) {
....
#endif
    } else if (strcmp(filesystemtype, "smartfs") == 0) {
...
    }

Reply:
Modified. Thank you!